File: C# OOP/Polymorphism-Lab/03.Shapes/Rectangle.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Shapes
{
    public class Rectangle : Shape
    {
        public Rectangle(double height, double width)
        {
            Height = height;
            Width = width;
        }

        public double Height { get; private set; }

        public double Width { get; private set; }


        public override double CalculateArea()
        {
            double result = Height * Width;

            return result;
        }

        public override double CalculatePerimeter()
        {
            double result = 2 * Height + 2 * Width;

            return result;
        }
        public override string Draw()
        {
            return $"{base.Draw()} {GetType().Name}";
        }

    }
}

File: Java OOP/Polymorphism-Lab/src/shapes/Rectangle.java
package shapes;

public class Rectangle extends Shape implements RectangleInterface{
    private Double height;
    private Double width;

    public Rectangle(Double height, Double width) {
        setHeight(height);
        setHeight(width);
    }

    @Override
    Double calculatePerimeter() {
        return 2 * height + 2 * width;
    }

    @Override
    Double calculateArea() {
        return height * width;
    }

    public void setHeight(Double height) {
        this.height = height;
    }

    public void setWidth(Double width) {
        this.width = width;
    }

    @Override
    public Double getHeight() {
        return height;
    }

    @Override
    public Double getWidth() {
        return width;
    }
}
